beacon-chain/forkchoice/doubly-linked-tree/store.go

```diff
@@ -148,11 +150,23 @@ func (s *Store) insert(ctx context.Context,
 			return err
 		}
 	}
-
 	// Update metrics.
 	processedBlockCount.Inc()
 	nodeCount.Set(float64(len(s.nodeByRoot)))
 
+	// Apply proposer boost
```

Comment:
I didn't feel like adding a helper since this is called only once and would take a lot of parameters that are available at this call site.